src/components/toast/tests/Toast.spec.tsx

```diff
@@ -0,0 +1,227 @@
+      let newToastAttributes = _.extend({}, toastBasicAttributes, { type: ToastType.Error });
+
+      expect(toastComponent.find(expectedClass).length).toBe(0);
+
+      toastComponent.setProps(newToastAttributes);
+      toastComponent.mount();
+      expect(toastComponent.find(expectedClass).length).toBe(1);
+    });
+
+    it('should have class "mod-animated" if the animate props is undefined or true', () => {
+      let expectedClass = '.mod-animated';
+      let newToastAttributes = _.extend({}, toastBasicAttributes, { animate: true });
+
+      expect(toastComponent.find(expectedClass).length).toBe(1);
+
+      toastComponent.setProps(newToastAttributes);
+      toastComponent.mount();
+      expect(toastComponent.find(expectedClass).length).toBe(1);
+
+      newToastAttributes = _.extend({}, toastBasicAttributes, { animate: false });
+      toastComponent.setProps(newToastAttributes);
+      toastComponent.mount();
+      expect(toastComponent.find(expectedClass).length).toBe(0);
+    });
+
+    it('should have a description when the content is set', () => {
+      const descriptionContainer = '.toast-description';
+      const expectedDescription = 'description';
+      let newToastAttributes = _.extend({}, toastBasicAttributes, { content: expectedDescription });
+
+      expect(toastComponent.find(descriptionContainer).length).toBe(0);
+
+      toastComponent.setProps(newToastAttributes);
+      toastComponent.mount();
+      expect(toastComponent.find(descriptionContainer).length).toBe(1);
+      expect(toastComponent.find(descriptionContainer).text()).toBe(expectedDescription);
+    });
+
+    it('should allow JSX in the content', () => {
+      const descriptionContainer = '.toast-description';
+      const expectedDescription = 'description';
+      let newToastAttributes = _.extend({}, toastBasicAttributes, { content: () => <a href='#'>{expectedDescription}</a> });
+
+      expect(toastComponent.find(descriptionContainer).length).toBe(0);
+
+      toastComponent.setProps(newToastAttributes);
+      toastComponent.mount();
+      expect(toastComponent.find(descriptionContainer).length).toBe(1);
+      expect(toastComponent.find(descriptionContainer).text()).toBe(expectedDescription);
+    });
+
+    it('should call onClose when the user clicks on .toast-close', () => {
+      const closeSpy = jasmine.createSpy('onClose');
+      const closeSelector = '.toast-close';
+      let newToastAttributes = _.extend({}, toastBasicAttributes, { onClose: closeSpy });
+
+      toastComponent.find(closeSelector).simulate('click');
+      expect(closeSpy).toHaveBeenCalledTimes(0);
+
+      toastComponent.setProps(newToastAttributes);
+      toastComponent.mount();
+
+      toastComponent.find(closeSelector).simulate('click');
+      expect(closeSpy).toHaveBeenCalledTimes(1);
+    });
+  });
+
+  describe('<Toast /> with a dismiss timer', () => {
+    const onCloseToast = jasmine.createSpy('onClose');
+    const dimissDelay = 2000;
```

Comment:
There's a typo here

Reply:
Changed for `const dimissDelay = 2000 - 1;`, added a comment and removed the `+ 1` in all `dimissDelay + 1`